### DISCUSSION TITLE
How can I manage useLocalStorageValue within useEffect?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [react-hookz/web#1093] [Q&A]
I'm trying to do the following:
```js
 useEffect(() => {
    lsVal.set(layouts);
  }, [layouts]);
```

But it's not properly setting the localStorage value.

**Comment by xobotyi** [comment]
If oly we would have a magical ball and thus knew what is "improperly" 🙃

**Reply by CodyBontecou** [reply]
Aha sorry. It's working fine actually.
